src/compat.h

```diff
@@ -79,6 +79,8 @@ struct Compatibility_t
     bool multiplayer_pause_controls;
     bool fix_fairy_stuck_in_pipe;
     bool world_map_fast_move;
+	//1.3.7
+	bool fix_flamethrower_gravity
```

Comment:
Please replace each tab with 4 spaces. While you contribute to any project, you should follow the same code style of the project. For this project, the 4-spaces indenting is used.

A tip: enable the showing of whitespaces in your text editor to see the actual content (tabs and spaces).